src/assets/scripts/client/aircraft/FlightManagementSystem/RouteModel.js

```diff
@@ -793,7 +793,7 @@ export default class RouteModel extends BaseModel {
         let starLegModel;
 
         try {
-            starLegModel = new LegModel(routeString);
+            starLegModel = new LegModel(routeString.toUpperCase());
```

Comment:
I'd be careful about making a change like this.  What problem is this solving exactly?  Are there tests you could add to both reproduce the problem and also prove it has been solved?

String comparisons should ideally be done as lowercase, so if anything this should be `toLowerCase()` instead.

Reply:
Yeah, so this technically is not really solving it. Originally the STAR command does not work because when creating an instance of the LegModel (to apply the new STAR entered by the user), _determineLegType seems to not be able to find the STAR name when it's in lower case. This _technically_ fixes the issue but I understand I need to look more into trying and lowercasing stuff as #1173 says.